### PR TITLE
Centrifuge App: Pool Configuration: Update tranches and combine with Epoch

### PR DESCRIPTION
**Pull request by onnovisser**
### Description

Combine Epoch and Tranches section on Pool Configuration page and allows parts of the tranches to be edited

### Approvals

- [ ] Dev

### Screenshots

image

image

### Impact

Centrifuge App, CentrifugeJS

### CODE REVIEW

File: centrifuge-app/src/pages/IssuerPool/Configuration/EpochAndTranches.tsx [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+            {token.tokenName}
+          </Text>
+        </Shelf>
+      ),
+      flex: '3',
+    },
+    {
+      align: 'left',
+      header: 'Token symbol',
+      cell: (token: Row) => token.symbolName,
```
Comment (sophialittlejohn) [inline centrifuge-app/src/pages/IssuerPool/Configuration/EpochAndTranches.tsx:46]: I think you need to wrap the cells in a `<Text variant="body2">` component, the typography looks too big

**Review by sophialittlejohn — APPROVED** [review]
Besides the font that my be too big everything else looks good 🙌

File: centrifuge-app/src/pages/IssuerPool/Configuration/EpochAndTranches.tsx (hunk 1 above)
Comment (onnovisser) [reply centrifuge-app/src/pages/IssuerPool/Configuration/EpochAndTranches.tsx:46]: Ah yes, looks to be to big. Gonna change this in DataTable though I think, as `body2` seems to be the default

**Review by sophialittlejohn — COMMENTED** [review]
looks like the share summary is broken:
Screenshot 2022-09-27 at 14 36 48

**Comment by onnovisser** [comment]
@sophialittlejohn fixed the share summary

**Review by sophialittlejohn — APPROVED** [review]
🔥

**Pull request merged**